server.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express');
+
+const { getTheater, getMovieShowtimes } = require('./db/db.js');
+
+const app = express();
+const port = 9002;
+
+app.use(express.static('dist'));
+app.listen(port);
+
+app.get('/showtime/:title/:zip', (req, res) => {
+  const title = req.params.title.split('%20').join(' ');
+
+  const data = {
+    movie_title: title,
```

Comment:
@chrisweilacker 
Are you saying I should change my API from '/showtime/:title_url/:zip' to '/showtime/:id/:zip'.
Then inside my API '/showtime/:id/:zip', I should make a get request to your '/m/movieinfo/:id' to get title_url then get results from my own db?

Reply:
Chatted with Chris, I will refactor the movies schema to use movie title_url as the foreign key instead of movie id since that's the only information my service has about movies. In my service's API, it will call Chris' service to get the full movie info.